devfile-support/samples/sample-devfile.yaml

```diff
@@ -4,8 +4,11 @@ metadata:
 projects:
   - name: "my-project"
     git:
-      location: "https://github.com/che-incubator/devworkspace-api"
-      branch: "master"
+      remotes:
+        origin: "https://github.com/devfile/api"
+      checkoutFrom:
+        revision: "master"
+        remote: origin
```

Comment:
Is it mandatory to have the `origin` remote in the `checkoutFrom`, since there is only one origin ?

Reply:
it's not mandatory, it's just here for sample.